config/presets/testnet.go

```diff
@@ -56,6 +56,9 @@ func testnet() config.Config {
 	conf.POST.MaxNumUnits = 4
 	conf.POST.MinNumUnits = 2
 
+	conf.POET.CycleGap = 10 * time.Second
+	conf.POET.PhaseShift = 20 * time.Second
```

Comment:
it is better to remove poet config from this pr completely, it will only cause confusion since it is not used

Reply:
i was gonna use this in the wait computation and forgot (and used 35 instead...)